02.wo_Defects/deletion_of_data_structure_sentinel.c

```diff
@@ -62,7 +62,7 @@ void deletion_of_data_structure_sentinel_003()
 	memcpy(str,ptr,11);/*Tool should Not detect this line as error*/ /*No ERROR:Deletion of a data structure sentinel*/
 }
 
-extern int vflag;
+extern volatile int vflag;
```

Comment:
the original definition had volatile so every declaration needs it too otherwise you can't read the value without being undefined.